charts/ionos-exporter/templates/deployment.yaml

```diff
@@ -46,6 +48,21 @@ spec:
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
           env:
+            - name: CONFIG_CONTENT
```

Comment:
You have a reference to the configfile as a parameter and also as environment variable. Here I would prefer the parameter